src/jsonLogic.js

```diff
@@ -106,3 +106,33 @@ export function yupSchemaWithCustomJSONLogic({ field, validations, config, id })
       }
     );
 }
+
+export function calculateComputedAttributes(fieldParams, { parentID = 'root' } = {}) {
+  return ({ validations, formValues }) => {
+    const { name, computedAttributes } = fieldParams;
+    const attributes = Object.fromEntries(
+      Object.entries(computedAttributes)
+        .map(handleComputedAttribute(validations, formValues, parentID, name))
+        .filter(([, value]) => value !== null)
+    );
+
+    return attributes;
+  };
+}
+
+function handleComputedAttribute(validations, formValues, parentID, name) {
+  return ([key, value]) => {
+    if (key === 'const')
+      return [
+        key,
+        validations.getScope(parentID).evaluateComputedValueRuleForField(value, formValues, name),
+      ];
+
+    if (typeof value === 'string') {
```

Comment:
Just want to confirm - subsequent PRs (honestly have been waiting so long to use PR instead of MR again 😂 ) will include additional logic for this function, so that we do not need to handle other cases where `key !== 'const'` or `typeof value === 'string'` - is that correct? If it is, no worries. If not, then do we need to guard against values we don't recognize, or will it silently fail?

Reply:
Confirmed when I looked at Part 3. :)